src/BetterPhpDocParser/PhpDocParser/StaticDoctrineAnnotationParser.php

```diff
@@ -80,6 +86,24 @@ public function resolveAnnotationValue(
         ];
     }
 
+    public function getCommentFromRestOfAnnotation(
+        BetterTokenIterator $tokenIterator,
+        string $annotationContent
+    ): string {
+        // we skip all the remaining tokens from the end of the declaration of values
+        while (
+            preg_match(self::END_OF_VALUE_CHARACTERS_REGEX, $tokenIterator->currentTokenValue())
+        ) {
+            $tokenIterator->next();
+        }
+
+        // the remaining of the annotation content is the comment
+        $comment = substr($annotationContent, $tokenIterator->currentTokenOffset());
+        // we only keep the first line as this will be added as a line comment at the end of the attribute
+        $commentLines = explode("\n", $comment);
```

Comment:
windows new line need to be supported, you can do `Strings::split()` like in 

https://github.com/rectorphp/rector-src/blob/a4445df500e51deb893319e180c3fbdf64e8dc25/src/Console/Formatter/ColorConsoleDiffFormatter.php#L62

Reply:
For reference for `\r?\n` usage https://github.com/search?q=repo%3Arectorphp%2Frector-src%20%5Cr%3F%5Cn&type=code